core-domain/src/main/java/org/uniprot/core/unirule/Condition.java

```diff
@@ -14,4 +14,6 @@
     boolean isNegative();
```

Comment:
General comment:
Can you please add tests with only required fields.
And another simple test to add is the roundTrip tests, similar with what we do in json....  
The ideia is:
create full object --> toXml --> fromXml --> result 
assertEquals(object, result)
Choose the order that is easier for this test...